.github/workflows/ci-rc-test.yaml

```diff
@@ -30,192 +30,15 @@ defaults:
-          python3 dev/integration_test_scripts/replace_dependencies.py --issue-url $rc_issue_url
-
-      - name: Check repo providers updated
-        id: check-repo-provideres-updated
-        run: |
-          difference=`git diff`
-          if [ -z "$difference" ]
-          then
-            echo "No provider changed"
-            echo "no_prvider_changed=true" >> $GITHUB_OUTPUT
-          else
-            echo "$difference"
-          fi
-
-      - name: Create RC branch
-        id: create_rc_branch
-        run: |
-          if [ "${{ steps.check-repo-provideres-updated.outputs.no_prvider_changed }}" != "true" ]
-          then
-            current_timestamp=`date +%Y-%m-%dT%H-%M-%S%Z`
-            echo "Current timestamp is" $current_timestamp
-            branch_name="rc-test-$current_timestamp"
-            git checkout -b $branch_name
-
-          else
-            branch_name=""
-          fi
-          echo "rc_testing_branch=$branch_name"
-          echo "rc_testing_branch=$branch_name" >> $GITHUB_OUTPUT
-
-
-      - name: Commit changes and create a pull request
-        if: steps.create_rc_branch.outputs.rc_testing_branch != ''
-        env:
-          GH_TOKEN: ${{ github.token }}
-        run: |
-          rc_issue_url="${{ steps.export-rc-issue-url.outputs.rc_issue_url }}"
-
-          git add pyproject.toml
-          git commit -m "Updating pyproject.toml with RC provider packages on $rc_issue_url"
-          git push origin ${{ steps.create_rc_branch.outputs.rc_testing_branch }}
-          gh pr create --title "[DO NOT MERGE] Test RC astro-sdk packages for $rc_issue_url" \
-                       --fill
-
-          echo "git_rev=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT
-    outputs:
-      rc_testing_branch: ${{ steps.create_rc_branch.outputs.rc_testing_branch }}
-
-  export-rc-testing-branch-name:
-    needs: [validate-manual-input, create-branch-for-testing-rc-release]
-    if: |
-      always() &&
-      (
-        needs.create-branch-for-testing-rc-release.result == 'success'  &&
-        needs.create-branch-for-testing-rc-release.outputs.rc_testing_branch != ''
-      ) ||
-      (
-        needs.validate-manual-input.result == 'success' &&
-        inputs.rc_testing_branch
-      )
-    runs-on: 'ubuntu-20.04'
-    steps:
-      - name: export rc_testing_branch
-        id: export-rc-testing-branch-name-step
-        run: |
-          if [ "${{ inputs.rc_testing_branch }}" == "" ]; then
-            rc_testing_branch=${{ needs.create-branch-for-testing-rc-release.outputs.rc_testing_branch }}
-          else
-            rc_testing_branch=${{ inputs.rc_testing_branch }}
-          fi
-          echo "rc_testing_branch=$rc_testing_branch" >> $GITHUB_OUTPUT
-    outputs:
-      rc_testing_branch: ${{ steps.export-rc-testing-branch-name-step.outputs.rc_testing_branch }}
+  check-airflow-provider-rc-release:
+    uses: astronomer/astronomer-providers/.github/workflows/reuse-wf-check-rc-release.yaml@main
```

Comment:
wondering if we need to update https://github.com/astronomer/astronomer-providers/blob/main/dev/integration_test_scripts/replace_dependencies.py too since in astronomer-provider we update the dependency in setup.cfg but here we update in pyproject.toml?

Reply:
nope, it'll still use the dev/integration_test_scripts/replace_dependencies.py in astro-sdk. I think there's some room for deduplicate further. But I'm occupied by other tasks. Maybe we can get back to this later.